library/asn1parse.c

```diff
@@ -150,12 +150,28 @@ int mbedtls_asn1_get_int( unsigned char **p,
     int ret;
     size_t len;
 
+#if defined(MBEDTLS_ASN1_PARSE_NEGINT)
+    unsigned char *neg = 0;
+#endif /* MBEDTLS_ASN1_PARSE_NEGINT */
+
     if( ( ret = mbedtls_asn1_get_tag( p, end, &len, MBEDTLS_ASN1_INTEGER ) ) != 0 )
         return( ret );
 
-    if( len == 0 || len > sizeof( int ) || ( **p & 0x80 ) != 0 )
+    if( len == 0 || len > sizeof( int ) )
         return( MBEDTLS_ERR_ASN1_INVALID_LENGTH );
 
+    if ((**p & 0x80) != 0)
```

Comment:
Please obey the Mbed TLS coding style and use no space after `if` but a space after/before a starting/closing bracket.